src/site/content/en/blog/optimizing-content-efficiency-eliminate-downloads/index.md

```diff
@@ -23,9 +25,38 @@ The fastest and best-optimized resource is a resource not sent. You should elimi
+
+By questioning whether or not you need a specific resource, you're not just speeding up page loads, you're also reducing the cognitive effort necessary to preserve layout stability. That leads to not only a much less frustrating user experience, but a less frustrating developer experience, as you'll have more time to pursue other goals in your projects.
+
+### Interaction to Next Paint (INP) and First Input Delay (FID)
+
+[Interaction to Next Paint (INP)](/inp/) and [First Input Delay (FID)](/fid/) are metrics that measure responsiveness to user inputs. [While INP is slated to replace FID as a Core Web Vital](/inp-cwv/) in March of 2024, optimization strategies for FID tend to also apply to INP. Furthermore, INP is generally more difficult to optimize for than FID, as it tracks the full interaction latency for _all_ page interactions, not just the input delay of the first interaction as FID measures.
+
+INP and FID tend to be most affected by JavaScript, as JavaScript is what drives most of the interactivity one experiences across the web. For both INP and FID, the amount of script resources downloaded during page load will kick off potentially expensive work involved in [script evaluation and compilation](/script-evaluation-and-long-tasks/). The less JavaScript you load during startup, the less work the browser has to do at that critical point in the page experience.
+
+While there are strategies for reducing the _size_ of JavaScript resources downloaded during startup—such as code splitting and tree shaking—it's worth auditing the packages you use in your projects to see if they're necessary at all. For example, [lodash]() has many methods that are still useful today, but ships with methods that the browser provides out of the box, such as `Array`-specific functions for [mapping](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map), [reducing](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce), and [filtering](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), and [many others](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array).
+
+Progressive enhancement is also [a useful approach](/adaptive-serving-based-on-network-quality/) to JavaScript, as it enables you to serve a baseline (but still functional) experience for users that you can add to for users with more powerful devices and faster network connections. Whether you adhere to the principle of progressive enhancement or not, the point remains: Every JavaScript resource you can avoid downloading can result in an experience that responds faster to user interactions, which is a vital aspect of web performance.
```

Comment:
The article just kind ends. Another reason to move the Summary bullet points down (maybe with an introductiary sentence or two)?